tracpro/static/js/baseline.js

```diff
@@ -0,0 +1,29 @@
+$(function() {
+        $("input[class^='datepicker']").datepicker();
+
+        // Display/hide question if poll is selected
+        if (!$("#id_baseline_poll").val()) {
+            $("#id_baseline_question").parent().parent().hide();
+        }
+        $("#id_baseline_poll").on("change", function() {
+          if(this.value) {
+            $("#id_baseline_question").parent().parent().show();
+          }
+          else {
+            $("#id_baseline_question").parent().parent().hide();
+          }
+        });
+
+        // Display/hide question if poll is selected
+        if (!$("#id_follow_up_poll").val()) {
```

Comment:
You can remove some duplicated logic by calling `$("#id_follow_up_poll,#id_baseline_poll").change()` after hooking up the `'change'` behavior.